src/styles/popover.css

```diff
@@ -1,9 +1,8 @@
 .ce-popover {
   position: absolute;
   opacity: 0;
-  visibility: hidden;
```

Comment:
unfortunatelly, this breaks the animation. Try to open/close the popover several times.